zenith/src/main.rs

```diff
@@ -157,6 +157,35 @@ pub fn get_branch_timeline(repopath: &Path, branchname: &str) -> ZTimelineId {
     ZTimelineId::from_str(&(fs::read_to_string(&branchpath).unwrap())).unwrap()
 }
 
+fn get_branch_infos(env: &LocalEnv) -> Result<HashMap<ZTimelineId, String>> {
```

Comment:
A brief function comment to explain what this does and what the return value is would be nice